python/ql/test/experimental/CWE-074/TRender.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+from django.http import HttpResponse
+from trender import TRender
+
+urlpatterns = [
+    path('', trender)
+]
+
+
+def trender(request):
+    template = request.GET['template']
+    compiled = TRender(template)
+    return HttpResponse(compiled)
```

Comment:
That should at least make this code valid :wink: 

```suggestion
def trender(request):
    template = request.GET['template']
    compiled = TRender(template)
    return HttpResponse(compiled)
    
urlpatterns = [
    path('', trender)
]
```